api/src/main/java/io/strimzi/api/kafka/model/balancing/KafkaRebalanceState.java

```diff
@@ -21,12 +21,12 @@ public enum KafkaRebalanceState {
      * A proposal has been requested from Cruise Control, but is not ready yet.
      * Transitions to:
      * <dl>
-     *     <dt>PendingProposal</dt><dd>A rebalance proposal is not ready yet.</dd>
+     *     <dt>PendingProposal</dt><dd>A rebalance proposal is not ready yet or strimzi.io/rebalance=refresh is applied due to spec changes</dd>
```

Comment:
How do you know the `strimzi.io/rebalance=refresh` was added by you because of a spec change and not by the user because he wants a refresh?

Reply:
I think this comment still stands. How do you know the annotation is there due to spec change?

Reply:
I had a go through regarding this. I think if we are `refresh` to trigger this mechanism then its possible for the user to do it manually themselves. I can fix this documentation. Would that be fine or we don't want the user to use `refresh` in these state manually @ppatierno @kyguy @scholzj ?

Reply:
My comment was really only about the Javadoc that did not seem to be right.

Reply:
Ahh Okay. I will fix it